src/pages/Tournament/TournamentDetails/BasicInfoPanel/index.tsx

```diff
@@ -0,0 +1,2 @@
+export { default as BasicInfoPanel } from './InfoPanel';
```

Comment:
Codacy has a fix for the issue: [Replace `Panel·}·from·'./Info` with `EditPanel·}·from·'./Edit`](https://app.codacy.com/gh/othello-organisation-singapore/othello-storm-system-frontend/pullRequest?prid=8062249)
```suggestion
export { default as BasicInfoEditPanel } from './EditPanel';
```